src/react-intl-messages.ts

```diff
@@ -71,6 +71,8 @@ export enum FM {
     APP_HEADER_DOCS = 'App.header.docs',
     APP_HEADER_SUPPORT = 'App.header.support',
 
+    APP_VERSION = 'App.version',
```

Comment:
Technically this is MODEL_VERSION but everything else was named with APP_ prefix so I followed pattern to avoid confusion.  We renamed APP to MODEL a while back.